docs/headless.md

```diff
@@ -0,0 +1,108 @@
+# Headless Guide
+
+## Running Project Locally
+
+1. Activate store channel in the Channels Manager
+2. For local debugging, set `VITE_LOCAL_DEBUG` to `false` in .env.
+3. Make sure that the buyer portal is running `yarn dev`
+4. Add the buyer portal scripts to your headless storefront files running locally
+
+- Script for the `<head>` tag
+
+```html
+<script type="module">
+  import RefreshRuntime from 'http://localhost:3001/@react-refresh'
+  RefreshRuntime.injectIntoGlobalHook(window)
+  window.$RefreshReg$ = () => {}
+  window.$RefreshSig$ = () => (type) => type
+  window.__vite_plugin_react_preamble_installed__ = true
+</script>
+<script type="module" src="http://localhost:3001/@vite/client"></script>
+```
+
+- Script for the `<body>` tag and make sure to replace the `{REPLACE_WITH_YOUR_STORE_HASH}` and `{REPLACE_WITH_YOUR_CHANNEL_ID}` with the appropriate information
+
+```html
+<script type="module" src="http://localhost:3001/src/main.ts"></script>
+<script>
+  window.B3 = {
+    setting: {
+      store_hash: '{REPLACE_WITH_YOUR_STORE_HASH}',
+      channel_id: { REPLACE_WITH_YOUR_CHANNEL_ID },
```

Comment:
I think curly braces can cause confusion, so it should be better remove them
```suggestion
- Script for the `<body>` tag and make sure to replace the `REPLACE_WITH_YOUR_STORE_HASH` and `REPLACE_WITH_YOUR_CHANNEL_ID` with the appropriate information

```html
<script type="module" src="http://localhost:3001/src/main.ts"></script>
<script>
  window.B3 = {
    setting: {
      store_hash: 'REPLACE_WITH_YOUR_STORE_HASH',
      channel_id: REPLACE_WITH_YOUR_CHANNEL_ID,
```